io.openems.edge.controller.ess.fixactivepower/test/io/openems/edge/controller/ess/fixactivepower/EssFixActivePowerImplTest.java

```diff
@@ -48,15 +45,18 @@ public void testOff() throws OpenemsException, Exception {
 						.setMode(Mode.MANUAL_OFF) //
 						.setHybridEssMode(HybridEssMode.TARGET_DC) //
 						.setPower(1234) //
-						.build()) //
-				.next(new TestCase() //
-						.output(ESS_SET_ACTIVE_POWER_EQUALS, null));
+						.setPhase(Phase.ALL) //
+						.setRelationship(Relationship.EQUALS) //
+						.build()); //
 	}
 
 	@Test
 	public void testGetAcPower() throws OpenemsException, Exception {
```

Comment:
A new test for Relationship and a different Phase configuration would be great, but otherwise everything is fine.